rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2019-02-03
```

Comment:
Have you tried leaving out this line?

Reply:
Leaving out this line leads to compile errors due to unstable features used in rocket.